server/task.js

```diff
@@ -0,0 +1,11 @@
+var db = require("./connection");
+
+var task = db.model('mytasks', {
```

Comment:
Maybe just call the model "tasks" instead of "mytasks"? 😄Who knows if they are really mine? 😉